mllib/src/main/scala/org/apache/spark/mllib/tree/impl/DTStatsAggregator.scala

```diff
@@ -73,26 +73,31 @@ private[spark] class DTStatsAggregator(
    * Flat array of elements.
    * Index for start of stats for a (feature, bin) is:
    *   index = featureOffsets(featureIndex) + binIndex * statsSize
-   * Note: For unordered features,
-   *       the left child stats have binIndex in [0, numBins(featureIndex) / 2))
-   *       and the right child stats in [numBins(featureIndex) / 2), numBins(featureIndex))
    */
   private val allStats: Array[Double] = new Array[Double](allStatsSize)
 
+  /**
+   * Array of parent node sufficient stats.
```

Comment:
Add note: "This is necessary because stats for the parent node are not available on the first iteration of tree learning."